corehq/apps/domain/models.py

```diff
@@ -449,6 +449,9 @@ class Domain(QuickCachedDocumentMixin, BlobMixin, Document, SnapshotMixin):
     ga_opt_out = BooleanProperty(default=False)
     orphan_case_alerts_warning = BooleanProperty(default=False)
 
+    # For domains that have been migrated to a different environment
+    redirect_url = StringProperty()
```

Comment:
> Adds yet another property to the Domain model, but this was deemed better than creating a new model for just one string field.

No objections here, but I do have some passing idle thoughts.  I have also wondered how best to deal with these random one-off domain settings.  The obvious one would be a `DomainSettings` (sql) model that's got a bunch of columns on it.  There are few enough domains that we don't really need to worry about the scale of data stored.

Another possibility that occurs to me, especially for sparsely populated fields is to use an entity-attribute-value model that doesn't require a migration for new fields.  Something like:
```python
class DomainSetting(models.Model):
    domain = models.CharField(db_index=True)
    slug = models.CharField(db_index=True)
    value = models.HmmActuallyMaybeThisIsHarderThanIThought()
```
Only thing I don't like about that is you'd have to use one fixed schema for `value`, which might be a string or a boolean or something else, depending on the setting.  Okay maybe a simple `DomainSettings` model _is_ best.

Reply:
675f785d3c4

Reply:
if we are creating a new model, how about exposing it to django admin?

Reply:
43efa33b1b9